src/phoenix/server/app.py

```diff
@@ -325,8 +325,9 @@ def create_app(
         msg = (
             "\n\n⚠️⚠️ Phoenix failed to migrate the database to the latest version. ⚠️⚠️\n\n"
             "The database may be in a dirty state. To resolve this, the Alembic CLI can be used\n"
-            "from the `src/phoenix/db` directory inside the Phoenix project root. From here,\n"
-            "revert any partial migrations and run `alembic stamp` to reset the migration state,\n"
+            "from the `src/phoenix/db` directory inside the Phoenix project root. First, clone\n"
+            "the Phoenix repo: https://github.com/Arize-ai/phoenix. From here, revert any\n"
+            "partial migrations and run `alembic stamp` to reset the migration state,\n"
```

Comment:
I'm guessing most users will need help to know how to revert partial migrations.

Reply:
I'm not sure these instructions are ultimately helpful as you will have to checkout the tag of the version they are running. Might make more sense to link to a help doc